src/main/java/com/cleanroommc/groovyscript/compat/mods/tinkersconstruct/Alloying.java

```diff
@@ -13,8 +17,10 @@
 import java.util.Arrays;
 import java.util.List;
 
+@RegistryDescription
 public class Alloying extends VirtualizedRegistry<AlloyRecipe> {
 
+    @RecipeBuilderDescription(example = @Example(".fluidOutput(fluid('iron') * 3).fluidInput(fluid('clay') * 1,fluid('lava') * 2).register()"))
```

Comment:
remove the `.register()` in here and apply the `@RecipeBuilderRegistrationMethod` annotation to the `register` method in the RecipeBuilder. this will generate the proper wiki text, and is required to get the the return types.

Reply:
Should i do this for all classes?

Reply:
yup